packages/report-server/src/reportBuilder/transform/functions/aggregate/aggregations.ts

```diff
@@ -95,6 +95,21 @@ const last = (values: FieldValue[]): FieldValue => {
   return values.slice().reverse().find(isNotUndefined);
 };
 
+const single = (values: FieldValue[]): FieldValue => {
```

Comment:
I was pretty temped to call this `only` instead. I think it really depends on the order of the config. Since we go `column: aggregation` I preferred `BCD1: single` over `BCD1: only`